tools/querytee/proxy.go

```diff
@@ -39,6 +39,7 @@ type ProxyConfig struct {
 	UseRelativeError                    bool
 	PassThroughNonRegisteredRoutes      bool
 	SkipRecentSamples                   time.Duration
+	SkipSamplesBefore                   int64
```

Comment:
Is there a particular reason you chose to use a Unix epoch timestamp rather than say a `time.Time`?

Reply:
Only reason I can think is I was lazy to write the time parsing lines :). Maybe it is a good idea to accept the time as a `RFC3339` string.